release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1102,6 +1102,62 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
+
+* Previously,  when a cluster failed to install on an on-premise platform with a configured load balancer, the `LoadBalancer` service's `LoadBalancerReady` condition received the `SyncLoadBalancerFailed` status. The status would trigger the generation of the `The kube-controller-manager logs might contain more details.` message. This message is wrong because the logs are stored in the `cloud-controller-manager` namespace of a project. With this release,  the `SyncLoadBalancerFailed` status now communicates the correct message: `The cloud-controller-manager logs may contain more details`. (link:https://issues.redhat.com/browse/OCPBUGS-31664[*OCPBUGS-31664*])
+
+* Previously, you could not control log levels for the internal component that selects IP addresses for cluster nodes. With this release, you can now enable debug log levels so that you can either increase or decrease log levels on-demand. To adjust log levels, you must create a config map manifest file with a configuration analogous to the following:
++
+[source,yaml]
+----
+apiVersion: v1
+data:
+  enable-nodeip-debug: "true"
+kind: ConfigMap
+metadata:
+  name: logging
+  namespace: openshift-vsphere-infra
+# ...
+----
++
+(link:https://issues.redhat.com/browse/OCPBUGS-32348[*OCPBUGS-32348*])
+
+* Previously, the Ingress Operator could not successfully update the canary route because the Operator did not have permission to update `spec.host` or `spec.subdomain` on an existing route. With this release, the required permission is added to the cluster role for the Operator’s service account and the Ingress Operator can update the canary route. (link:https://issues.redhat.com/browse/OCPBUGS-36465[*OCPBUGS-36465*])
+
+* Previously, administrator privileges were required to run some networking containers, such as Keepalived, on supported on-premise platforms. With this release, these containers no longer require administrator privileges to run them on supported on-premise platforms. (link:https://issues.redhat.com/browse/OCPBUGS-36175[*OCPBUGS-36175*])
+
+* Previously, if your `NodeNetworkConfigurationPolicy` (NNCP) custom resource (CR) is set to use the default spanning tree protocol (STP) implementation, the CR YAML file would show `stp.enabled: true`, but the {product-title} web console deselected the STP checkbox. This situation is misleading. With this release, the web console only deselects the STEP checkbox after you define `stp.enabled: false` in the NNCP CR YAML file.
+(link:https://issues.redhat.com/browse/OCPBUGS-36238[*OCPBUGS-36238*])
+
+* Previously, the ingress controller status was incorrectly displayed as `Degraded=False` because of a migration time issue with the `CanaryRepetitiveFailures` condition. With this release, the ingress controller is correctly marked as `Degraded=True` for the appropriate length of time that the `CanaryRepetitiveFailures` condition exists. (link:https://issues.redhat.com/browse/OCPBUGS-39220[*OCPBUGS-39220*])
```

Comment:
```suggestion
* Previously, the Ingress Controller status incorrectly displayed as `Degraded=False` because of a migration time issue with the `CanaryRepetitiveFailures` condition. With this release, the Ingress Controller status is correctly marked as `Degraded=True` for the appropriate length of time that the `CanaryRepetitiveFailures` condition exists. (link:https://issues.redhat.com/browse/OCPBUGS-39220[*OCPBUGS-39220*])
```

Reply:
:heavy_check_mark: